tests/fixtures/rpm-ostree-status.json

```diff
@@ -0,0 +1,60 @@
+{
+  "deployments" : [
+    {
+      "unlocked" : "none",
+      "requested-local-packages" : [
+      ],
+      "base-commit-meta" : {
+        "coreos-assembler.config-dirty" : "true",
+        "coreos-assembler.config-gitrev" : "e5ffda727170e124c7f7d8782e256b0462bf9869",
+        "coreos-assembler.basearch" : "x86_64",
+        "rpmostree.inputhash" : "0e2ba98bac847c8a8cc5ffd42822a41d53ef6719ccaf6a0094559768aa24da33",
+        "version" : "30.1",
+        "rpmostree.rpmmd-repos" : [
+          {
+            "id" : "fedora-coreos-pool",
+            "timestamp" : 7435743550895030272
+          },
+          {
+            "id" : "fedora",
+            "timestamp" : -7689964138319052800
```

Comment:
@jlebon not strictly related to this PR, but I fear there something going wrong in the compose if this timestamp ends up negative.

Reply:
Yeah, that's tracked in https://github.com/projectatomic/rpm-ostree/issues/1734. Need to look at that again soon.